source/mainnet/net/guides/recover-wallet.rst

```diff
@@ -0,0 +1,58 @@
+.. include:: ../../variables.rst
+.. _recover-wallet:
+
```

Comment:
The screenshots on this page look a little weird. It looks like there should be a frame because of the rounded corners, but it doesn't seem to be there.